_maps/map_files/Deltastation/DeltaStation2_skyrat.dmm

```diff
@@ -168635,7 +168635,7 @@ bnI
 ciP
 bgZ
 clA
-cns
+clA
```

Comment:
how did the change do it only one lettert

Reply:
skill